raiden/api/v1/resources.py

```diff
@@ -121,3 +122,17 @@ def get(self, **kwargs):
             kwargs['from_block'],
             kwargs['to_block']
         )
+
+
+class TransferToTargetResource(BaseResource):
+
+    post_schema = TransferSchema(
+        exclude=('initiator_address', 'target_address', 'token_address')
+    )
+
+    def __init__(self, **kwargs):
+        super(TransferToTargetResource, self).__init__(**kwargs)
+
+    @use_kwargs(post_schema, locations=('json',))
+    def post(self, **kwargs):
```

Comment:
as discussed in the other PR I totallly agree. But I think it's only here and in the other PR I ever did it, so both will be fixed now.